### ISSUE TITLE
Batch normalisation

### ISSUE THREAD

**Issue by hallvardnmbu** [hallvardnmbu/neurons#23]

**Comment by hallvardnmbu** [comment]
Might not be that simple to implement, as forward pass currently is applied to one input at a time (non-batched); batch inputs are processed parallel.

[wikipedia](https://en.wikipedia.org/wiki/Batch_normalization)
[blog-post](https://www.deepchecks.com/glossary/batch-normalization/)

**Comment by hallvardnmbu** [comment]
It would need relatively large refactoring.
